ntoskrnl/config/cmsysini.c

```diff
@@ -844,6 +849,8 @@ NTAPI
 INIT_FUNCTION
 CmpInitializeSystemHive(IN PLOADER_PARAMETER_BLOCK LoaderBlock)
 {
+    static const WCHAR HivePath[] = L"\\SystemRoot\\System32\\Config\\SYSTEM";
```

Comment:
I understand that deduplicating it is an improvement. Not sure I agree that moving the string away from the code that uses it is.

Reply:
Moving this string here does not deduplicate anything. It was taken from the patch on JIRA.
I can move that back if you like.

Reply:
I agree with moving it back. I guess the reason I moved it here was that it used to be duplicated, but the 2nd instance went away.

Reply:
Fixed in latest commit